src/Build.UnitTests/Evaluation/Evaluator_Tests.cs

```diff
@@ -4326,7 +4328,8 @@ public void VerifyDTDProcessingIsDisabled2()
                 Thread.Sleep(500);
 
                 // Expect server to be alive and hung up unless a request originating from DTD processing was sent
-                Assert.True(t.IsAlive);
+                _httpListenerThreadException.ShouldBeNull();
```

Comment:
is this check necessary? `_httpListenerThreadException` is not being set anywhere in this test

Reply:
It may be set indirectly: this test fires off a thread running `HttpServerThread`, which may set the exception if it fails in some bad way (this is how I found the problem: for some reason it couldn't open the port on my machine).

Reply:
Ah I missed that.